assets/app/styles/_core.less

```diff
@@ -104,36 +97,24 @@ html, body {
       margin-bottom: 0;
     }
   }
-}
-@media (min-width: @screen-sm-min) {
-  .console-os .navbar-project {
-    .navbar-project-menu {
-      display: inline-block;
-      float: left;
-      width: @navbar-project-menu-sm;
-    }
-    .navbar-search {
-      margin-left: @sidebar-left-width-open-sm;
-    }
+  .active-filters {
+    /* If line-height is set, the active-filters div takes up space even
+       when no filters are active. */
+    line-height: 0;
   }
-}
-@media (min-width: @screen-lg-min) { 
-  .console-os .navbar-project {
-    .navbar-project-menu {
-      width: @navbar-project-menu-md;
-    }
-    .navbar-search {
-      margin-left: @sidebar-left-width-open-md;
-    }
+  .create-button-container {
+    margin-top: 15px;
   }
 }
-@media (min-width: @screen-xlg-min)  {
+@media (min-width: @screen-sm-min) {
   .console-os .navbar-project {
-    .navbar-project-menu {
-      width: @navbar-project-menu-lg;
+    .navbar-filter-widget {
+      /* Prevent the filter button from wrapping at wide screen sizes. */
+      min-width: 340px;
     }
-    .navbar-search {
-      margin-left: @sidebar-left-width-open-lg;
+    .create-button-container {
+      text-align: right;
+      margin-top: 30px;
```

Comment:
add a comment about what this is for... to align the button with things that have a top-label